web_development_101/installations/setting_up_git.md

```diff
@@ -147,6 +147,7 @@ Now, go back to GitHub in your browser window and paste the key you copied into
 #### Step 2.5 Testing your key
 
 Follow the directions in [this article from GitHub](https://help.github.com/en/articles/testing-your-ssh-connection) to verify your SSH connection. If the output doesn't correctly match up, then try going through these steps again or come to [the Discord chat](https://discord.gg/hvqVr6d) to ask for help. 
+Note: Do not inlcude the `$` when typing in the commands given in the article. 
```

Comment:
Is this the first lesson where commands are presented with a `$`?

If so, it would be good to point out what the `$` is or the purpose it serves in conjunction with that it should not be included in the commands.